quickstart/docker/image/add-router.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
```

Comment:
i'd recommend you just update the big, ziti-cli-functions.sh if needed. I think much/most maybe all the things you're wanting to do already exists as functions in that script. for example there is already a `createEdgeRouterConfig "${ZITI_EDGE_ROUTER_RAWNAME}"`